modules/local/seqera_runs_dump/functions.nf

```diff
@@ -1,6 +1,14 @@
-
 @Grab('com.github.groovy-wslite:groovy-wslite:1.1.2;transitive=false')
 import wslite.rest.RESTClient
+import groovy.json.JsonSlurper
+
+// Set system properties for custom Java trustStore
+def setTrustStore(trustStorePath, trustStorePassword) {
+    System.setProperty("javax.net.ssl.trustStore", trustStorePath)
+    if (trustStorePassword) {
+        System.setProperty("javax.net.ssl.trustStorePassword", trustStorePassword)
+    }
+}
```

Comment:
is this needed to access Platform API ?

Reply:
Yes, if your Platform uses a private certificate. In this case, we need to configure the REST client to use a trust store containing the private certificate and allow the client to establish secure https connections back to the platform.